compact/far_forward.xml

```diff
@@ -0,0 +1,18 @@
+<!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
+<!-- Copyright (C) 2022 Wouter Deconinck, Sakib Rahman -->
+
+<lccdd>
+
+  <include ref="far_forward/ion_beamline.xml" />
+  <include ref="far_forward/beampipe_hadron_B0.xml" />
+  <include ref="far_forward/electron_beamline.xml" />
+  <include ref="far_forward/B0_tracker.xml"/>
+  <include ref="far_forward/B0_ECal.xml"/>
+  <include ref="far_forward/offM_tracker.xml"/>
+  <include ref="far_forward/ZDC_SiPMonTile.xml"/>
+  <include ref="far_forward/ZDC_Crystal_LYSO.xml"/>
+  <include ref="far_forward/roman_pots_eRD24_design.xml"/>
+  <include ref="far_forward/vacuum.xml"/>
+  <include ref="far_forward/magnets.xml"/>
+
+</lccdd>
```

Comment:
This file should remain deleted, per https://github.com/eic/epic/pull/679. Make any changes to `compact/far_forward/default.xml`.